modules/imgproc/test/test_goodfeaturetotrack.cpp

```diff
@@ -62,123 +62,28 @@ struct greaterThanPtr
     { return *a > *b; }
 };
 
-static void
```

Comment:
why is the reference implementation removed? please, restore and use it

Reply:
I removed it because I was also testing the response value which was not accurate with that test method.
I am restoring it right now